src/daemon/metric.c

```diff
@@ -66,7 +66,10 @@ static label_pair_t *label_set_read(label_set_t labels, char const *name) {
     return NULL;
   }
 
-  label_t label = {
+  struct {
+    char const *name;
+    char const *value;
+  } label = {
```

Comment:
Why is this explicit instead of `label_pair_t`?

Reply:
`name` is a `char const *` and can't be assigned to `(label_pair_t).name`, which is a `char *` (i.e. not const). I preferred to keep the argument `const`. Should I add a comment to clarify?

Reply:
That is fine, just wondering.